driver-core/src/test/functional/com/mongodb/internal/operation/ChangeStreamOperationSpecification.groovy

```diff
@@ -227,7 +227,7 @@ class ChangeStreamOperationSpecification extends OperationFunctionalSpecificatio
         nextDoc.getFullDocument() == BsonDocument.parse('{ _id : 2, x : 3 }')
         nextDoc.getNamespace() == helper.getNamespace()
         nextDoc.getOperationType() == OperationType.UPDATE
-        nextDoc.getUpdateDescription() == new UpdateDescription(['y'], BsonDocument.parse('{x : 3}'))
+        nextDoc.getUpdateDescription() == new UpdateDescription(['y'], BsonDocument.parse('{x : 3}'), null)
```

Comment:
It is not specified whether the MongoDB server returns `null` or `[]` when no changes are reported as truncation. The current behavior appears to be returning `[]`. The method `UpdateDescription.equals` treats `null` and empty `List` as equal values, this way we are prepared to face any of `null`/`[]`, and can safely specify `null` in this test as our expectation.

Reply:
Well from released servers its null but from now on it maybe an empty list

Reply:
Yes, it may be an empty list since MongoDB 4.7, or a null. The server design does not specify that one of these values is not possible, and we cannot depends on the behavior we currently observe: this same design is an example of a situation when the server [changed its behaviour because it was not specified](https://docs.google.com/document/d/1sxGrck8X0r0W37UVLFwV8vaUQv3OM8xlyWWvHB5M2ew/edit#heading=h.22yp33tzmzpp).

Reply:
Ack